bin/import.ml

```diff
@@ -30,8 +30,21 @@ include Common.Let_syntax
 
 (* FIXME: leverage fibers to actually connect in the background *)
 let make_memory () =
+  let handle = function
+    | Dune_manager.Client.Dedup (target, source) -> (
+      let target = Path.Build.to_string target in
+      let tmpname = target ^ ".dedup" in
```

Comment:
We should use something that's out of the way of what the user may see, i.e. a path in `_build/.dedup` for instance.